packages/progress-indicator/translations/en.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  loading: 'Loading',
```

Comment:
The accessibility readers will see this component as "Loading", which is probably correct in nearly all cases, but there may be places where a progress indicator is used that does not indicate "Loading" but something else. Do we want to make it possible to override this?

Reply:
Makes sense. We could consider calling a method `_setLabel()` inside `onLocaleUpdated`, so Sub classers can override this